icu4j/main/classes/core/src/com/ibm/icu/text/BreakIteratorFactory.java

```diff
@@ -135,11 +135,18 @@ private static BreakIterator createBreakInstance(ULocale locale, int kind) {
             if ( lbKeyValue != null && (lbKeyValue.equals("strict") || lbKeyValue.equals("normal") || lbKeyValue.equals("loose")) ) {
                 typeKeyExt = "_" + lbKeyValue;
             }
+            if(typeKeyExt == null) {
+                String lwKeyValue = locale.getKeywordValue("lw");
+                if("phrase".equals(lwKeyValue)) {
```

Comment:
I think it is better 
```
if(lwKeyValue.equals("phrase")) {
```

Reply:
I think 
"phrase".equals(lwKeyValue) is shorter than
lwKeyValue != null && lwKeyValue.equals("phrase")